src/os/ui/window/arcserverhelp.js

```diff
@@ -0,0 +1,79 @@
+goog.module('os.ui.window.ArcServerHelpUI');
```

Comment:
Plugin-specific code should be contained within the plugin. Please move this under `src/plugin/arc` and update the module name appropriately.